src/settings-ui/Settings.UI/Flyout/LaunchPage.xaml.cs

```diff
@@ -54,13 +52,11 @@ private void ModuleButton_Click(object sender, RoutedEventArgs e)
                 case "Hosts": // Launch Hosts
                     {
                         bool launchAdmin = SettingsRepository<HostsSettings>.GetInstance(new SettingsUtils()).SettingsConfig.Properties.LaunchAdministrator;
-                        var actionName = "Launch";
-                        if (!App.IsElevated && launchAdmin)
+                        string eventName = App.IsElevated || !launchAdmin ? Constants.ShowHostsSharedEvent() : Constants.ShowHostsAdminSharedEvent();
```

Comment:
I'm puzzled by this condition `App.IsElevated || !launchAdmin`. Should it be launchAdmin (without !)?

Reply:
I understand that it's really puzzling but it should be fine.
Reverted to AND condition to improve readability.

```
string eventName = !App.IsElevated && launchAdmin
    ? Constants.ShowHostsAdminSharedEvent()
    : Constants.ShowHostsSharedEvent();
```

- If PT is not elevated but launch hosts elevated is on: launch elevated
- Else: launch normal